### PR TITLE
Governance Page UI

### PR DESCRIPTION
**Pull request by AtelyPham**
- feat: update new tabs deisgn
- style: mixer page to match new tab design
- style: bridge page to match new tab design
- style: vbridge page to match new tab design
- style: rounded tab
- style: wrap/unwrap page to match new tab design
- refactor: applied className for tab icon
- feat: governance package and tab on side bar
- feat: add proposal card component
- feat: list proposal and pagination setup
- feat: setup pagination and loading state
- chore: center title and update height of loading component for consistent UI

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Currently, we don't have the governance page.

Issue Number: https://github.com/webb-tools/webb-dapp/issues/308

## What is the new behavior?

I have implemented the Governance Page UI (demo below). But currently, we are using mockup data for display information.

https://user-images.githubusercontent.com/60747384/177410791-53e898c6-4e76-417e-a6aa-34c43862ed1e.mp4

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
Currently, I'm using the `TablePagination` component from `material-ui/core v4`, we need to migrate `v5` to use the `Pagination` component.

### CODE REVIEW

File: packages/react-hooks/src/useCountDown.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+
+/**
+ *
+ * @param targetDate the target date to count down from
+ * @returns an object contains `days`, `hours`, `minutes`, `seconds` and `isExpired` caculate from `targetDate`
```
Comment (dutterbutter) [inline packages/react-hooks/src/useCountDown.ts:6]:
```suggestion
 * @param targetDate: Represents the target date to count down from
 * @returns an object containing `days`, `hours`, `minutes`, `seconds` and `isExpired` calculated from `targetDate`
```

File: packages/react-hooks/src/useCountDown.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      setCountDown(countDownDate - new Date().getTime());
+    }, 1000);
+
+    return () => clearInterval(interval);
+  }, [countDownDate]);
+
+  /**
+   *
+   * @param value the value to check for add leading `0`
+   * @returns the value with leading `0` if value less than `10`
```
Comment (dutterbutter) [inline packages/react-hooks/src/useCountDown.ts:24]:
```suggestion
   * @param value: Represents the value to check for adding the leading `0`
   * @returns the value with leading `0` if value is less than `10`
```

File: packages/react-hooks/src/useCountDown.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    return value < 10 ? '0' + value.toString() : value.toString();
+  };
+
+  return { ...getReturnValues(countDown), zeroPad };
+};
+
+/**
+ *
+ * @param countDown the count down value to calculate time left
+ * @returns an object contains `days`, `hours`, `minutes`, `seconds` and `isExpired` caculate from `targetDate`
```
Comment (dutterbutter) [inline packages/react-hooks/src/useCountDown.ts:36]:
```suggestion
 * @param countDown: Represents the count down value to calculate time left
 * @returns an object containing `days`, `hours`, `minutes`, `seconds` and `isExpired` calculated from `targetDate`
```

**Review by dutterbutter — CHANGES_REQUESTED** [review]
Looks great so far @AtelyPham !!

**Comment by dutterbutter** [comment]
Can you increase the spacing between results and pagination. 

Screen Shot 2022-07-08 at 8 51 23 AM

**Review by dutterbutter — APPROVED** [review]

**Review by nepoche — APPROVED** [review]
Looks great!

**Pull request merged**